core/domain/feedback_services.py

```diff
@@ -75,7 +97,22 @@ def create_message(
     """Creates a new message for the thread and subscribes the author to the
     thread.
 
-    Returns False if the message with the ID already exists.
+    Args:
+        exploration_id: str. The exploration id the thread belongs to.
+        author_id: str. The author id who creates this message.
+        updated_status: str, one of STATUS_CHOICES. New thread status.
+            Must exist in the first message of a thread. For the
+            rest of the thread, should exist only when the status changes.
+        updated_subject: str. New thread subject. Must exist in the first
+            message of a thread. For the rest of the thread, should exist
+            only when the subject changes.
+        text: str. The text of the message being created which is allowed not
+            to exist.
+        has_suggestion: bool. Whether this thread has a related
+            learner suggestion.
+
+    Returns:
+        bool. If returns normally, it must be True.
```

Comment:
Hm, the return value seems a bit superfluous. Is it ever used? If not, maybe we should get rid of it...